models/models.py

```diff
@@ -48,14 +156,15 @@ def get_pareto_routes(df, start_time, end_time, bar_num, total_max_walking_time,
     :return: A list of Solutions
     """
     solutions = []
-    interval_length = 5
-    for max_time in range(0, int(total_max_walking_time), interval_length):
-        solutions.append(get_optimal_route(df, start_time, end_time, bar_num, max_time, max_walking_each))
-    pass
+    for max_walking_time in range(30, int(total_max_walking_time*60), 5):
+        for max_waiting_time in range(30, int(max_total_wait*60), 5):
```

Comment:
I don't think we need both dimensions for the pareto chart. We should probably determine what the optimal decision variables should be (e.g. walking time vs rating, etc.). I feel rating is logical to have, but would be open to wait or walk time for the other.